profile.d/toolbox.sh

```diff
@@ -20,9 +20,10 @@ eval $(
 if [ -f /run/ostree-booted ] \
    && ! [ -f "$host_welcome_stub" ] \
    && [ "${ID}" = "fedora" ] \
-   && { [ "${VARIANT_ID}" = "workstation" ] || [ "${VARIANT_ID}" = "silverblue" ]; }; then
+   && { [ "${VARIANT_ID}" = "workstation" ] || [ "${VARIANT_ID}" = "silverblue" ] || [ "${VARIANT_ID}" = "kinoite" ]; }; then
     echo ""
-    echo "Welcome to Fedora Silverblue. This terminal is running on the"
+    # shellcheck disable=SC3059
+    echo "Welcome to Fedora ${VARIANT_ID^}. This terminal is running on the"
```

Comment:
This sort of case modification with a `^` is a Bash-ism.  It doesn't work with Z shell.  See https://github.com/containers/toolbox/issues/1017 or try this:
```
% . /etc/os-release
% echo "${VARIANT_ID^}"
zsh: bad substitution
```

So, yes, ShellCheck was right.